content/sensu-go/6.7/observability-pipeline/observe-schedule/agent.md

```diff
@@ -1147,6 +1148,19 @@ sensu-agent start --log-level debug{{< /code >}}
 /etc/sensu/agent.yml example | {{< code shell >}}
 log-level: debug{{< /code >}}
 
+<a id="max-session-length-attribute"></a>
+
+| max-session-length |      |
+--------------|------
+description   | Maximum duration for any one agent connection. In milliseconds (`ms`), seconds (`s`), minutes (`m`), or hours (`h`). When a connection reaches the specified amount of time, Sensu will force the agent to disconnect and reconnect to an available configured backend to prevent agent connection distribution from becoming skewed over time. The max-session-length algorithm includes random jitter so that agents will not disconnect and reconnect all at once.
```

Comment:
To clarify, it's called the "maximum amount of time" in that it is an upper bound. There will be a reconnection _before_ that duration is reached, not _when_ that duration is reached.

The exact random jitter is an implementation detail, but currently it is calculated so that the reconnection can happen any time between `[0.5 * max, max]` ([code here](https://github.com/sensu/sensu-go/blob/ba2acff99c824d33f911f583ab60df75130a8f70/agent/agent.go#L503-L526)).

So I would suggest changing the wording from "When a connection reaches the specified amount of time" to something like "Some time before a connection reaches the specified amount of time"?